manyfews/calculations/zentra.py

```diff
@@ -0,0 +1,238 @@
+from .models import ZentraReading
+from .models import ZentraDevice
+from zentra.api import *
```

Comment:
Could you specify which classes you use from here, e.g. `ZentraReadings`? It will make the code below clearer.

Reply:
accepted and changed.